.kitchen.yml

```diff
@@ -18,5 +19,7 @@ platforms:
 
 suites:
   - name: default
-    run_list:
-      - recipe[test::default]
+    verifier:
```

Comment:
This is the default value and can be removed.

Reply:
ok